sci2zero/TeslaRIS-backend
# Feature/assessment

## Description

Added initial assessment model as well as basic initial CRUD for all entities. Added view and download statistic aggregation to test the soundness of the implementation.

## How should this be tested?

This should be tested in pair with [frontend PR](https://github.com/sci2zero/TeslaRIS-frontend/pull/36). When logged in as an administrator user, you should be able to perform basic CRUD operations on assessment entities. Statistics should be viewable with any access level.

## Checklist
